### PR TITLE
Backport Cliquet docs

### PR DESCRIPTION
**Pull request by leplatrem**
Supposed to ease future backporting :)

### CODE REVIEW

**Comment by Natim** [comment]
Are you sure do you want to do this modification in readinglist ?

File: docs/resource.rst [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+.. _resource-endpoints:
 
 GET /articles
 =============
 
-**Requires an FxA OAuth authentication**
+**Requires authentication**
 
-Returns all articles of the current user.
+Returns all records of the current user for this resource.
```
Comment (Natim) [inline docs/resource.rst]: Can we keep articles?

**Comment by leplatrem** [comment]
My idea (suggestion) here was to backport as much docs as possible from Cliquet.
This way, when we upgrade Cliquet, the diff on docs is much smaller...

File: docs/resource.rst (hunk 1 above)
Comment (leplatrem) [inline docs/resource.rst]: I don't see why mentionning records would be a problem. If we keep articles, then this PR is not so relevant :)

**Pull request merged**